src/bin/pg_autoctl/cli_get_set_properties.c

```diff
@@ -92,17 +102,21 @@ CommandLine get_commands =
 static CommandLine set_node_replication_quorum_command =
 	make_command("replication-quorum",
 				 "set replication-quorum property on the monitor",
-				 CLI_PGDATA_USAGE "<true|false>",
-				 CLI_PGDATA_OPTION,
-				 cli_getopt_pgdata,
+				 " [ --pgdata ] [ --json ] [ --name ] <true|false>",
```

Comment:
Seems like this one still needs `--formation`

Reply:
Fixed.